pom.xml

```diff
@@ -117,12 +117,13 @@
             <version>${websocket.api.version}</version>
         </dependency>
         <!-- test dependencies -->
-        <dependency>
+        <!-- testkit no available for since 5.0.0-RC6 -->
```

Comment:
Could you please clarify this line - so we're sure we understand it in the future?

> testkit no available for since 5.0.0-RC6